lib/reek/configuration/configuration_file_finder.rb

```diff
@@ -31,7 +31,7 @@ def find_by_cli(options)
 
       def find_by_dir(start)
         start.ascend do |dir|
-          files = dir.children.select(&:file?)
+          files = dir.children.select(&:file?).sort
```

Comment:
I don't understand how this change relates to:

>> This fixes a small problem I had where removing exceptions.reek and then adding it again broke the ConfigurationFileFinder specs.

Could you please explain that a little more detailed?